### DISCUSSION TITLE
Check out our latest release! 0.2 Beta

### DISCUSSION THREAD

**Discussion by dayland** [microsoft/PubSec-Info-Assistant#135] [Show and tell]
This week we have launched our [0.2 Beta release](https://github.com/microsoft/PubSec-Info-Assistant/releases/tag/v0.2-Beta). It has some great new features that we think you can really take advantage of. 

A huge thanks to all our 0.1 Alpha consumers who tested and provided feedback to help make the 0.2 Beta release possible. 

@mick199
@infinitelyloopy-bt
@jcst0
@flickturner
@kapilmehtagit
@Lachy70
@akdataai
@yomorgan
@gkoundour73
@lmwilki
@geostation360
@demokedes
@Donste13
@kousikExterro
@jayaraj-Kumar
@stephenadam5
@dakotad88
@hendrickct
@katouziam
@kawlil
@kckc-Leidos
@MackBlackburn
@ningyu07
@Robertwadelinger
@Shiflettj
@sprinkler
@velazquej1
@stephenadam5
@brunicholson
@jh-pa-dev
@dnsaldi
@kellyvehent
@fitchtravis
@SPGaud
@pocket44
@woojin444
@neaorin
@ferlageckin
@RezaMadi
@vickyarp
@meinradweiss
@christopherwoodland
@lon-tierney

**Comment by yomorgan** [comment]
[heart]         Yonathan Morgan reacted to your message:
________________________________
From: dayland ***@***.***>
Sent: Monday, July 17, 2023 7:25:46 PM
To: microsoft/PubSec-Info-Assistant ***@***.***>
Cc: Yonathan Morgan ***@***.***>; Mention ***@***.***>
Subject: [microsoft/PubSec-Info-Assistant] Check out our latest release! 0.2 Beta (Discussion #135)


This week we have launched our 0.2 Beta release<https://github.com/microsoft/PubSec-Info-Assistant/releases/tag/v0.2-Beta>. It has some great new features that we think you can really take advantage of.

A huge thanks to all our 0.1 Alpha consumers who tested and provided feedback to help make the 0.2 Beta release possible.

@Mick199<https://github.com/Mick199>
@infinitelyloopy-bt<https://github.com/infinitelyloopy-bt>
@jcst0<https://github.com/jcst0>
@flickturner<https://github.com/flickturner>
@kapilmehtagit<https://github.com/kapilmehtagit>
@Lachy70<https://github.com/Lachy70>
@akdataai<https://github.com/akdataai>
@yomorgan<https://github.com/yomorgan>
@gkoundour73<https://github.com/gkoundour73>
@lmwilki<https://github.com/lmwilki>
@geostation360<https://github.com/geostation360>
@demokedes<https://github.com/demokedes>
@Donste13<https://github.com/Donste13>
@kousikExterro<https://github.com/kousikExterro>
@jayaraj-Kumar<https://github.com/jayaraj-Kumar>
@stephenadam5<https://github.com/stephenadam5>
@dakotad88<https://github.com/dakotad88>
@hendrickct<https://github.com/hendrickct>
@katouziam<https://github.com/katouziam>
@kawlil<https://github.com/kawlil>
@kckc-Leidos<https://github.com/kckc-Leidos>
@MackBlackburn<https://github.com/MackBlackburn>
@ningyu07<https://github.com/ningyu07>
@Robertwadelinger<https://github.com/Robertwadelinger>
@shiflettj<https://github.com/shiflettj>
@sprinkler<https://github.com/sprinkler>
@velazquej1<https://github.com/velazquej1>
@stephenadam5<https://github.com/stephenadam5>
@brunicholson<https://github.com/brunicholson>
@jh-pa-dev<https://github.com/jh-pa-dev>
@Dnsaldi<https://github.com/Dnsaldi>
@kellyvehent<https://github.com/kellyvehent>
@fitchtravis<https://github.com/fitchtravis>
@SPGaud<https://github.com/SPGaud>
@pocket44<https://github.com/pocket44>
@woojin444<https://github.com/woojin444>
@neaorin<https://github.com/neaorin>
@ferlageckin<https://github.com/ferlageckin>
@RezaMadi<https://github.com/RezaMadi>
@vickyarp<https://github.com/vickyarp>
@meinradweiss<https://github.com/meinradweiss>
@christopherwoodland<https://github.com/christopherwoodland>

**Comment by lon-tierney** [comment]
Checking in to see if anyone is having issues with installation of the Beta? If so, please reach out on this thread or email us at isat-support@microsoft.com
